plugins/modules/system/sap_snote.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com> <rainer.leber@sva.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: sap_snote
+
+short_description: This module will upload and (de)implements C(SNOTES) in a SAP S4HANA environment.
+
+version_added: "0.2.0"
+
+description:
+    - The C(sap_snote) module depends on C(pyrfc) Python library (version 2.4.0 and upwards).
+        Depending on distribution you are using, you may need to install additional packages to
+        have these available.
+    - This module will use the Function Group C(SCWB_API).
+    - The C(TMS) must be configured at first.
+    - The user could not be C(DDIC) or C(SAP).
```

Comment:
Dont know what this means. Needs explanation later on...

Reply:
DDIC and SAP are the default Admins in The System but these users are Not able to do everything like Transports

Reply:
We should think about filter these users and catch them as an error ?